src/main/java/aeneas/models/Level.java

```diff
@@ -24,6 +27,25 @@ public Level(Bullpen bullpen, boolean prebuilt) {
   public Level(Bullpen bullpen) {
     this(bullpen, true);
   }
+  
+  @Override
+  public int hashCode() {
+    return levelNumber;
```

Comment:
What? Since I moved the stars earned to a HashMap<Level,Integer>, we need to implement hashCode, and levelNumber seems a pretty reasonable choice for a hash since there shouldn't be duplicates.

I could see arguments against moving out the stars earned, but I think we really want that data separate from the level itself.